utils/qemu_persistent_hook/read_into_rdi.c

```diff
@@ -5,7 +5,7 @@
 
 void afl_persistent_hook(struct x86_64_regs *regs, uint64_t guest_base,
                          uint8_t *input_buf, uint32_t input_buf_len) {
-
+\
```

Comment:
wrong blackslash

Reply:
Removed, it looks like the auto formatting might be adding this in?